content/docs/faq.md

```diff
@@ -35,11 +35,11 @@ By itself, Scratch Addons is just an addon loader. Its main tasks are:
 
 ### Is Scratch Addons safe? 
 
-Yes. Scratch Addons should not have any security issues/vulnerability in its most recent version, found in the Chrome Web Store and addons.mozilla.org. They have been verified by reviewers on contributors of Scratch Addons and by the extension stores.
+Yes. Scratch Addons is completely open source. Code is constantly being verified by Scratch Addons contributors, as well as by reviewers from the Chrome Web Store and Add-ons for Firefox.
```

Comment:
I think we shouldn't remove "should not have any security issues/vulnerabilities." Contrary to popular belief, just because it's open source, doesn't mean it's safe.

Reply:
Maybe we should simply remove `Yes.`

Reply:
I don't think that's a good idea. Starting with a short answer is already fine.

Also, "The code" instead of "Code?"